zerver/openapi/zulip.yaml

```diff
@@ -16113,6 +16113,11 @@ components:
           description: |
             The description of the hotspot, as will be displayed to the
             user.
+        has_trigger:
+          type: boolean
+          description: |
+            The has_trigger is used to identify if the hotspot will activate
+            when the specific event occurs.
```

Comment:
This needs an API feature level bump and `**Changes**` documentation, like all API changes.